src/main/src/test/java/org/geoserver/wfsng/WfsCompatibilityTest.java

```diff
@@ -67,12 +67,19 @@ public void testWfsCompatitibility() throws IOException {
         }
                 
         assertTrue(store instanceof WFSDataStore);
-                
-        FeatureType type = ftInfo.getFeatureType();
         
-        assertEquals("sf_archsites", type.getName().getLocalPart());        
+        try{
```

Comment:
Still not following formatting rules, it should be "try {" and "} catch(...) {", no need to add extra white lines in the block instead